students/10th/yeajinpark/login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+      <div class = "background" >
+        <img src = "img/logo_text.png">
+
+        <form class = "input-wrap">
+          <input class = "userinfo" type = "text" placeholder = "전화번호, 사용자 이름 또는 이메일">
+          <input class = "userinfo" type = "password" placeholder = "비밀번호">
+          <button> 로그인 </button>
```

Comment:
```suggestion
          <button>로그인</button>
```

불필요한 공백이 들어 있네요. 사소해보이지만 이런 습관 하나 하나가 좋은 코드를 작성하는데 도움이 됩니다 :)

Reply:
넵 공백 딜리트하겠습니다.